docs/configuration.html

```diff
@@ -100,7 +100,7 @@ <h5>Updating Password Configs in ZooKeeper Before Starting Brokers</h5>
   on broker 0:
 
   <pre class="brush: bash;">
-  &gt; bin/kafka-configs.sh --zookeeper localhost:2181 --entity-type brokers --entity-name 0 --alter --add-config
+  &gt; bin/kafka-configs.sh --bootstrap-server localhost:9092 --entity-type brokers --entity-name 0 --alter --add-config
```

Comment:
This specific change should not be made because the paragraph above is referring specifically to configuration "before starting brokers for bootstrapping".  However, since it is talking about storing sensitive information, it is likely that this would be a TLS-enabled Zookeeper quorum.  So I would suggest:
```suggestion
  &gt; bin/kafka-configs.sh --zookeeper localhost:2182 --zk-tls-config-file zk_tls_config.properties --entity-type brokers --entity-name 0 --alter --add-config
```

Reply:
Updated. Thanks.